lib/fake_ifdh.py

```diff
@@ -41,6 +41,20 @@
 DEFAULT_ROLE = "Analysis"
 
 
+def init_scitokens() -> None:
+    cfgfile = f'{os.environ.get("HOME", "/tmp")}/.jobsub_scitokens.cfg'
+    uid = os.getuid()
+    if not os.access(cfgfile, os.R_OK):
+        with open(cfgfile, "w") as cff:
+            cff.write(
+                f"[scitokens]\ncache_location: /var/run/user/{uid}/jobsub_scitokens.cache"
```

Comment:
Two things:

1.  The cache_location is supposed to be a directory where the DB lives, not the file itself (see the table here:  https://github.com/scitokens/scitokens#configuration)

2.  Just to make this more readable for future us, what are your thoughts about taking advantage of python's multiline strings and making this write statement:

```python
cff.write((
    "[scitokens]\n"
    f"cache_location: /var/run/user/{uid}\n"
))
```
(I added a newline at the end of the `cache_location` designation just o make the file look cleaner if someone `cat`s it in the future).

Reply:
I like the new edit.  Did you think we should do something like point (2) here, or do you think it's unnecessary?